### ISSUE TITLE
Responsive design of 'boxes'?

### ISSUE THREAD

**Issue by Jeroen-GroeneBij** [tomchadwin/qgis2web#535]
Hello,

If I am correct, the generated html is only partly responsive. The map itself does scale nicely when viewed on a mobile phone, tablet, laptop of big desktop screen. The 'boxes' such as Titel, Address search, zoom buttons, layers list or legend don't seem to be responsive to the screen size.
Particularly on mobile phones these boxes fill the little screen.
I only tested this with Leaflet.

Is there a way to make them responsive also, or to improve their scaling?

**Comment by tomchadwin** [comment]
I'm not sure, to be honest. A lot of the controls are generated by Leaflet plugins, so I'm not sure the best way to go about fixing them.

**Comment by Jeroen-GroeneBij** [comment]
I did ask a similar question in the Leaflet repository.
[https://github.com/Leaflet/Leaflet/issues/5754#event-1234414195](url)

The answer was, it is not really a leaflet issue but up the application (qgis2web) to make choices.
Main tip was to remove the line|:
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
I did try this and for me that is a usable answer. The boxes are much smaller now on a mobile phone screen, but not to small in my opinion.
